_redirects

```diff
@@ -75,3 +75,18 @@ https://babel.netlify.com/* https://babeljs.io/:splat 301!
 /docs/plugins/undeclared-variables-check/ /docs/en/babel-plugin-undeclared-variables-check
 
 /docs/en/next/tools/*               /setup
+
+/7.11.0 /blog/2020/07/30/7.11.0
+/7.10.0 /blog/2020/05/25/7.10.0
+/7.9.0 /blog/2020/03/16/7.9.0
+/7.8.0 /blog/2020/01/11/7.8.0
+/7.7.0 /blog/2019/11/05/7.7.0
+/7.6.0 /blog/2019/09/05/7.6.0
+/7.5.0 /blog/2019/07/03/7.5.0
+/7.4.0 /blog/2019/03/19/7.4.0
+/7.3.0 /blog/2019/01/21/7.3.0
+/7.2.0 /blog/2018/12/03/7.2.0
+/7.1.0 /blog/2018/09/17/7.1.0
+/7.0.0 /blog/2018/08/27/7.0.0
+/6.23.0 /blog/2017/02/13/6.23.0
+/6.23.3 /blog/2017/02/13/6.23.0
```

Comment:
Are we gonna doing the same for `7.10.5`, etc.?

Reply:
I guess we could, I only did this because of the versions page though.

Reply:
there is something called splats which you could try https://docs.netlify.com/routing/redirects/redirect-options/#splats

`/news/*  /blog/:splat`